examples/adafruit_io_simpletest_data.py

```diff
@@ -12,11 +12,11 @@
 # Import Adafruit IO REST Client
 from adafruit_io.adafruit_io import RESTClient, AdafruitIO_RequestError
 
-# Get wifi details and more from a wifi_settings.py.py file
+# Get wifi details and more from a esp32spi_secrets.py.py file
```

Comment:
just 'secrets.py' please - that way it matches pyportal's expectations!

Reply:
Ok, only secrets now - was looking at ESP32 at the time.